flow.js

```diff
@@ -160,11 +160,11 @@ class Flow {
         nextStep: [
         {condition: 'floorTiles', nextStep: 'unable-to-book'},
         {condition: 'skirtingBoardArchitrave', nextStep: 'unable-to-book'},
-        {condition: 'wallTiles', nextStep: 'unable-to-book'},
+        {condition: 'wallTiles', nextStep: 'repair-description'},
```

Comment:
Have these been added to the backend (third step in [this doc](https://docs.google.com/document/d/1wS8Lb-Zz2ZVuHAPi4xn3cxnIcWNuWZwQhbblceMC9s8/edit))